`cargo install` fails with errors about thread safety of OnceCell.

We currently cannot install Hipcheck with the `cargo install --path hipcheck` command. When this command is run, we get the error message:

```
error[E0277]: `std::cell::OnceCell<std::string::String>` cannot be shared between threads safely
   --> hipcheck/src/shell/progress_phase.rs:19:16
    |
19  | static STYLES: OnceLock<[ProgressStyle; 2]> = OnceLock::new();
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `std::cell::OnceCell<std::string::String>` cannot be shared between threads safely
    |
    = help: within `[ProgressStyle; 2]`, the trait `std::marker::Sync` is not implemented for `std::cell::OnceCell<std::string::String>`
    = note: if you want to do aliasing and mutation between multiple threads, use `std::sync::OnceLock` instead
note: required because it appears within the type `indicatif::state::TabExpandedString`
   --> /home/mchernicoff/.cargo/registry/src/index.crates.io-6f17d22bba15001f/indicatif-0.17.10/src/state.rs:354:17
    |
354 | pub(crate) enum TabExpandedString {
    |                 ^^^^^^^^^^^^^^^^^
note: required because it appears within the type `indicatif::style::TemplatePart`
   --> /home/mchernicoff/.cargo/registry/src/index.crates.io-6f17d22bba15001f/indicatif-0.17.10/src/style.rs:634:6
    |
634 | enum TemplatePart {
    |      ^^^^^^^^^^^^
note: required because it appears within the type `PhantomData<indicatif::style::TemplatePart>`
   --> /home/mchernicoff/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/marker.rs:753:12
    |
753 | pub struct PhantomData<T: ?Sized>;
    |            ^^^^^^^^^^^
note: required because it appears within the type `alloc::raw_vec::RawVec<indicatif::style::TemplatePart>`
   --> /home/mchernicoff/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:76:19
    |
76  | pub(crate) struct RawVec<T, A: Allocator = Global> {
    |                   ^^^^^^
note: required because it appears within the type `std::vec::Vec<indicatif::style::TemplatePart>`
   --> /home/mchernicoff/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:397:12
    |
397 | pub struct Vec<T, #[unstable(feature = "allocator_api", issue = "32838")] A: Allocator = Global> {
    |            ^^^
note: required because it appears within the type `indicatif::style::Template`
   --> /home/mchernicoff/.cargo/registry/src/index.crates.io-6f17d22bba15001f/indicatif-0.17.10/src/style.rs:466:8
    |
466 | struct Template {
    |        ^^^^^^^^
note: required because it appears within the type `ProgressStyle`
   --> /home/mchernicoff/.cargo/registry/src/index.crates.io-6f17d22bba15001f/indicatif-0.17.10/src/style.rs:21:12
    |
21  | pub struct ProgressStyle {
    |            ^^^^^^^^^^^^^
    = note: required because it appears within the type `[ProgressStyle; 2]`
    = note: required for `OnceLock<[ProgressStyle; 2]>` to implement `std::marker::Sync`
    = note: shared static variables must have a type that implements `Sync`

error[E0277]: `std::cell::OnceCell<std::string::String>` cannot be shared between threads safely
   --> hipcheck/src/shell/spinner_phase.rs:18:23
    |
18  | static SPINNER_STYLE: OnceLock<ProgressStyle> = OnceLock::new();
    |                       ^^^^^^^^^^^^^^^^^^^^^^^ `std::cell::OnceCell<std::string::String>` cannot be shared between threads safely
    |
    = help: within `ProgressStyle`, the trait `std::marker::Sync` is not implemented for `std::cell::OnceCell<std::string::String>`
    = note: if you want to do aliasing and mutation between multiple threads, use `std::sync::OnceLock` instead
note: required because it appears within the type `indicatif::state::TabExpandedString`
   --> /home/mchernicoff/.cargo/registry/src/index.crates.io-6f17d22bba15001f/indicatif-0.17.10/src/state.rs:354:17
    |
354 | pub(crate) enum TabExpandedString {
    |                 ^^^^^^^^^^^^^^^^^
note: required because it appears within the type `indicatif::style::TemplatePart`
   --> /home/mchernicoff/.cargo/registry/src/index.crates.io-6f17d22bba15001f/indicatif-0.17.10/src/style.rs:634:6
    |
634 | enum TemplatePart {
    |      ^^^^^^^^^^^^
note: required because it appears within the type `PhantomData<indicatif::style::TemplatePart>`
   --> /home/mchernicoff/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/marker.rs:753:12
    |
753 | pub struct PhantomData<T: ?Sized>;
    |            ^^^^^^^^^^^
note: required because it appears within the type `alloc::raw_vec::RawVec<indicatif::style::TemplatePart>`
   --> /home/mchernicoff/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:76:19
    |
76  | pub(crate) struct RawVec<T, A: Allocator = Global> {
    |                   ^^^^^^
note: required because it appears within the type `std::vec::Vec<indicatif::style::TemplatePart>`
   --> /home/mchernicoff/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:397:12
    |
397 | pub struct Vec<T, #[unstable(feature = "allocator_api", issue = "32838")] A: Allocator = Global> {
    |            ^^^
note: required because it appears within the type `indicatif::style::Template`
   --> /home/mchernicoff/.cargo/registry/src/index.crates.io-6f17d22bba15001f/indicatif-0.17.10/src/style.rs:466:8
    |
466 | struct Template {
    |        ^^^^^^^^
note: required because it appears within the type `ProgressStyle`
   --> /home/mchernicoff/.cargo/registry/src/index.crates.io-6f17d22bba15001f/indicatif-0.17.10/src/style.rs:21:12
    |
21  | pub struct ProgressStyle {
    |            ^^^^^^^^^^^^^
    = note: required for `OnceLock<ProgressStyle>` to implement `std::marker::Sync`
    = note: shared static variables must have a type that implements `Sync`

For more information about this error, try `rustc --explain E0277`.
```

--- a/hipcheck/Cargo.toml
+++ b/hipcheck/Cargo.toml
@@ -70,7 +70,7 @@ hipcheck-macros = { path = "../hipcheck-macros", version = "0.3.1" }
 http = "1.2.0"
 indexmap = "2.7.0"
 indextree = "4.7.3"
-indicatif = { version = "0.17.9", features = ["rayon"] }
+indicatif = { version = "0.17.11", features = ["rayon"] }
 itertools = "0.13.0"
 jiff = "0.1.16"
 kdl = "6.2.2"

